Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -3671,7 +3786,7 @@ auto B3IRGenerator::addDelegateToUnreachable(ControlType& target, ControlType& d
     if (ControlType::isTry(target))
         targetDepth = target.tryDepth();
 
-    m_exceptionHandlers.append({ HandlerType::Delegate, data.tryStart(), ++m_callSiteIndex, 0, m_tryCatchDepth, targetDepth });
+    m_exceptionHandlers.append({ HandlerType::Delegate, data.tryStart(), advanceCallSiteIndex(), 0, m_tryCatchDepth, targetDepth });
```

Comment:
Are we collecting exception handlers of inlined callee?

Reply:
Also can you review other fields to ensure that we are collecting them? Next week, we should make IRBuilder instance one, and we should separate data structures to CompileCallFrame to ensure this and do this cleanly.

Reply:
Yes